lib/rmt/config.rb

```diff
@@ -36,4 +36,28 @@ def self.mirror_src_files?
     ActiveModel::Type::Boolean.new.cast(Settings.try(:mirroring).try(:mirror_src))
   end
 
+  class << self
```

Comment:
All other methods in this class are `self.foo`. Maybe we could enclose them all in this block?